ui/src/app/pages/data/cohort-review/query-report.component.tsx

```diff
@@ -207,16 +210,14 @@ export const QueryReport = withCurrentWorkspace()(
     }
 
     componentDidMount() {
-      const {cdrVersionId} = this.props.workspace;
+      const {cdrVersionTiersResponse, workspace: {accessTierShortName, cdrVersionId}} = this.props;
       const {review} = this.state;
       const {ns, wsid, cid} = urlParamsStore.getValue();
       cohortsApi().getCohort(ns, wsid, cid).then(cohort => this.setState({cohort}));
-      cdrVersionsApi().getCdrVersions().then(resp => {
-        const cdrName = resp.items.find(
-          v => v.cdrVersionId === review.cdrVersionId.toString()
-        ).name;
-        this.setState({cdrName});
-      });
+      const cdrName = cdrVersionTiersResponse.tiers
```

Comment:
`findCdrVersion()` is also available if you prefer: https://github.com/all-of-us/workbench/blob/bff7743a8baf8b6a9162dc5ae211b48eb9b16dc7/ui/src/app/utils/cdr-versions.tsx#L21

Reply:
Good call, switched.